### ISSUE TITLE
error when resizing terminals inside tilix with specifically fish

### ISSUE THREAD

**Issue by T0mi10** [fish-shell/fish-shell#8691]
version: 3.1.2
os: Linux pop-os 5.15.15-76051515-generic #202201160435~1642693824~21.10~97db1bb SMP Thu Jan 20 17:35:05 U x86_64 x86_64 x86_64 GNU/Linux
TERM: xterm-256color
Problem:
When resizing terminals inside tilix with specifically fish (tested with `sh` and `bash` and neither crash), an assertion failed is thrown.
This only happens wheen the terminal is smal.
The error i got:
**
VTE:ERROR:../src/vteaccess.cc:628:void vte_terminal_accessible_text_scrolled(VteTerminal*, gint, gpointer): assertion failed: (i >= 1)
Bail out! VTE:ERROR:../src/vteaccess.cc:628:void vte_terminal_accessible_text_scrolled(VteTerminal*, gint, gpointer): assertion failed: (i >= 1)
fish: “tilix” terminated by signal SIGABRT (Abort)

video:

https://user-images.githubusercontent.com/83451390/152144532-dd00732b-b744-4d1d-a5b3-2c5adafe2a14.mov

**Comment by faho** [comment]
>version: 3.1.2

Fish 3.1.2 is five releases out of date. It is missing 20% of all commits ever made to fish. It is very much no longer supported.

If you want to report issues here, please upgrade. Unfortunately the Ubuntu package (which popos uses as well) is essentially unmaintained, so you can use [our PPA](https://launchpad.net/~fish-shell/+archive/ubuntu/release-3).

----

The thing that is crashing is your terminal. The "VTE" errors come from VTE, the underlying terminal library. Fish interacts with it differently, which causes it to no longer understand the situation and bail out by hitting an assert. It shouldn't do that. Asserts are there to specify a situation that "can't" happen. If they are hit, that's a bug and should be reported.


So, in summary:

1. Report this to your terminal's bug tracker instead
2. Update fish

**Comment by faho** [comment]
Okay, don't report it to your terminal either, this was fixed 10 months ago as https://gitlab.gnome.org/GNOME/vte/-/issues/207.

If anyone can do anything it's your distribution. These are the joys of LTS.
